src/dodal/devices/oav/oav_to_redis_forwarder.py

```diff
@@ -35,13 +37,10 @@ class Source(IntEnum):
 
 
 class OAVSource(StandardReadable):
-    def __init__(
-        self,
-        prefix: str,
-        oav_name: str,
-    ):
-        self.url = epics_signal_r(str, f"{prefix}MJPG_URL_RBV")
-        self.oav_name = oav_name
+    def __init__(self, oav: OAV, label: str):
+        self.url_ref = Reference(oav.grid_snapshot.url)
+        self.oav_ref = Reference(oav)
```

Comment:
Could: `oav_ref` is never used, just the `url_ref` is so I would remove `oav_ref`

Reply:
Hang on, I'm wrong, this is used in the `mx-bluesky` side. Let me review that side too and then approve this